src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+         int i = new Random().nextInt(Color.values().length);
+         return Color.values()[i];
```

Comment:
The return statement is attempting to return a `Color` enum constant, but the method is declared to return a `String`. Change the method's return type to `Color` to match the return value.